tests/it/execution.rs

```diff
@@ -196,8 +228,8 @@ fn can_nest_function_calls() {
 }
 
 #[test]
-fn skips_empty_matches() {
-    check_execution(
+fn cannot_use_nullable_regex() {
+    fail_parse(
```

Comment:
Is there a reason why this test of the parser doesn't live in `tests/it/parser.rs`?

Reply:
You are right, this should not be here. Let me fix that.